support-services/src/main/scala/com/gu/support/catalog/CatalogService.scala

```diff
@@ -14,8 +13,16 @@ object CatalogService {
 
 class CatalogService(val environment: TouchPointEnvironment, jsonProvider: CatalogJsonProvider) extends LazyLogging {
 
-  private[this] def getRatePlanId(billingPeriod: BillingPeriod, fulfilmentOptions: FulfilmentOptions)
-    = getProductRatePlan(environment, billingPeriod, fulfilmentOptions, NoProductOptions).map(_.id).getOrElse("")
+  def getProductRatePlan(
```

Comment:
This is to allow us to work out what product rate plan id to add into the SupporterProductData Dynamo table